script/Ciel/Day 5/05_01_CIEL05_1B.txt

```diff
@@ -87,64 +94,72 @@ The pair follow me inside, and one of them presses the B1 button.
 [sha:8211f651f5f0e2c050f954d5f94419d12e680b47]{
 -- Page 67, Offset 11036.
 -- 「……あの、どこに？」
-"...Um, to what floor?"
+"Um... where are we going?"
 }
 [sha:52b1d579415420e43a5f9070b3c9f8caa8763630]{
 -- Page 67, Offset 11037.
 -- 「地下だよ、地下。Ｂ１って見えねえか？　そこで一晩過ごしてくれればいい。戻ってきたら上客の仲間入りだ。丁重に扱ってやる」
-// 丁重に
-"We're headed to the basement, the basement. Don't you see the B1? That's where you should stay the night. And when you come back, you'll become a guest of honor. We'll roll the red carpet for you."
+"The basement, duh. %{i}B a s e m e n t%{/i}.
+ Can you not see where it says 'B1'?
+ That's where you'll be spending the night.
+ When you come back, you'll be a guest of honor.
+ We'll roll out the red carpet for you."
 }
 [sha:5e99a9669d581949cf2e82fea04bcaeacb4bd55a]{
 -- Page 68, Offset 11038.
 -- 「そうそう、すごーくカンタンでちゅからねー！　時間をつぶせばいいだけ、なぁーんにも怖いコトはありませーん！」
-"Yep yep, it's suuuper simple! You just need to stay there, there's nothing scary at all!"
+"Yep yep, it's %{i}suuuper%{/i} simple!
+ All you gotta do is stick around for a bit,
+ and there's nothing scary down there at all, no sir!"
```

Comment:
bit. There's not even anything scary down there, no sir!
bit. There's nothing scary down there at all, no sir!